pom.xml

```diff
@@ -16,18 +16,19 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- Dependencies version (external) -->
-    <org.apache.commons.commons-compress.version>1.22</org.apache.commons.commons-compress.version>
+    <org.apache.commons.commons-compress.version>1.25.0</org.apache.commons.commons-compress.version>
     <org.apache.commons.commons-lang3.version>3.12.0</org.apache.commons.commons-lang3.version>
     <commons-io.commons-io.version>2.11.0</commons-io.commons-io.version>
     <org.apache.commons.commons-email.version>1.5</org.apache.commons.commons-email.version>
     <org.freemarker.freemarker.version>2.3.32</org.freemarker.freemarker.version>
-    <com.google.guava.guava.version>31.1-jre</com.google.guava.guava.version>
-    <log4j.log4j.version>1.2.17</log4j.log4j.version>
+    <com.google.guava.guava.version>33.0.0-jre</com.google.guava.guava.version>
+    <log4j.log4j.version>2.22.1</log4j.log4j.version>
     <org.slf4j.version>1.7.30</org.slf4j.version>
+    <org.slf4j-reload4j.version>2.0.11</org.slf4j-reload4j.version>
     <org.slf4j.jcl-over-slf4j.version>1.7.30</org.slf4j.jcl-over-slf4j.version>
     <org.slf4j.jul-to-slf4j.version>1.7.30</org.slf4j.jul-to-slf4j.version>
     <junit.junit.version>4.13.2</junit.junit.version>
-    <com.h2database.h2.version>1.4.199</com.h2database.h2.version>
+    <com.h2database.h2.version>2.2.224</com.h2database.h2.version>
```

Comment:
This is going to break all existing instances using h2 database.

Reply:
Hi @jendib for h2 we have been working with this version of h2 in our forked teedy without any issues. However let me know and I can revert both this and the log4j version change. But there will still be vulnerabilities for these older versions